x-pack/plugins/aiops/server/types.ts

```diff
@@ -5,6 +5,16 @@
  * 2.0.
  */
 
+import { PluginSetup, PluginStart } from '@kbn/data-plugin/server';
+
+export interface AiopsPluginSetupDeps {
```

Comment:
Does this export, and the one below, contribute to the counts of public APIs which don't have comments?